src/components/Sum/Sum.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Sum = (props) => {
+  const { a, b } = props;
+
+  return <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>;
+};
```

Comment:
```suggestion
export const Sum = ({ a, b }) => {
  return <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>;
};
```
You can do it right here. Just a hint. :)